src/i18ndude/untranslated.py

```diff
@@ -243,60 +249,71 @@ def characters(self, data):
 
 
 class SilentHandler(Handler):
-
     def log(self, msg, severity):
         Handler.log(self, msg, severity)
 
-        if severity == 'FATAL':
-            self._out.write('Fatal error in document %s' % self._filename)
-            self._out.write('\n')
+        if severity == "FATAL":
+            self._out.write("Fatal error in document %s" % self._filename)
+            self._out.write("\n")
 
     def endDocument(self):
-        if not (self._stats['WARNING'] or self._stats['ERROR']):
+        if not (self._stats["WARNING"] or self._stats["ERROR"]):
             return
 
-        self._out.write('%s: %s warnings, %s errors'
-                        % (self._filename, self._stats['WARNING'],
-                           self._stats['ERROR']))
+        self._out.write(
+            "%s: %s warnings, %s errors"
+            % (self._filename, self._stats["WARNING"], self._stats["ERROR"])
+        )
 
 
 class VerboseHandler(Handler):
-
     def log(self, msg, severity):
         Handler.log(self, msg, severity)
 
-        self._out.write(unicode('%s:%s:%s:\n-%s- - %s\n' % (
-            self._filename,
-            self._parser.getLineNumber(),
-            self._parser.getColumnNumber(),
-            severity,
-            msg)))
-
-        if severity == 'FATAL':
-            char = '='
+        self._out.write(
+            str(
```

Comment:
Generally speaking, casting to string (here and in other places on this file) is not needed. Anyway, we can clean this in a new PR.

Reply:
I thought about changing this when making the PR, but worried that there might be cases where we would have `None` or maybe an integer as input, instead of bytes or str.  So I decided to leave it for now.